liblangutil/SourceReferenceFormatter.cpp

```diff
@@ -163,10 +173,15 @@ void SourceReferenceFormatter::printSourceLocation(SourceReference const& _ref)
 
 void SourceReferenceFormatter::printExceptionInformation(SourceReferenceExtractor::Message const& _msg)
 {
-	// exception header line
-	errorColored() << _msg.severity;
-	if (m_withErrorIds && _msg.errorId.has_value())
-		errorColored() << " (" << _msg.errorId.value().error << ")";
+    if (_msg.severity.eq("Error") && _msg.errorId.has_value())
```

Comment:
Why not `==`? It's just a `string`.
```suggestion
    if (_msg.severity == "Error" && _msg.errorId.has_value())
```

Reply:
Fixed 👍